### DISCUSSION TITLE
Copilot show suggestions but not autocompletes with [Tabs] in Markdown files

### DISCUSSION THREAD

**Discussion by dfg-98** [community/community#7861] [Copilot Conversations] [answered]
While editing markdown files Github Copilot shows great suggestions but when [Tab] is pressed for autocomplete a space is instered intead.

**Comment by hsenag** [comment, marked as answer]
The usual reason for this problem is a clash with another extension that works with markdown files. For example, [this blog post](https://dxrf.com/blog/2021/09/28/gh-copilot-markdown-all-in-one-extension/) describes a clash with "Markdown All in One".
